### PR TITLE
Upgrade tests to JUnit5

### PR DESCRIPTION
**Pull request by macenturalxl1**
Hi - JUnit5 changes for #188 - let me know what your thoughts are thanks

### CODE REVIEW

**Comment by macenturalxl1** [comment]
I've just noticed this from the pipeline logs:
`log4j:WARN No appenders could be found for logger (uk.gov.gchq.koryphe.ValidationResult).`

That doesn't seem to be logging the tests properly compared to a previous build so looking in to it now to resolve it - the tests are running and passing anyway

**Comment by macenturalxl1** [comment]
I've resolved the missing log4j config - I've used the exact same XML config used in Gaffer - now just trying to figure out why the tests aren't logging out as they have done inthe previous Pull Request build

File: pom.xml [hunk 1]
```diff
@@ -23,7 +23,8 @@
         <guava.version>27.0.1-jre</guava.version>
         <hamcrest.version>1.3</hamcrest.version>
         <jackson.version>2.6.5</jackson.version>
-        <junit.version>4.12</junit.version>
+        <!--        <junit.version>4.12</junit.version>-->
```
Comment (t11947) [inline pom.xml:26]: Could the commented lines in the pom be removed?

Comment (macenturalxl1) [reply pom.xml]: Thanks for spotting that - done

**Pull request merged**